src/dtgtk/thumbtable.c

```diff
@@ -878,8 +890,30 @@ static gboolean _event_scroll(GtkWidget *widget, GdkEvent *event, gpointer user_
 {
   GdkEventScroll *e = (GdkEventScroll *)event;
   dt_thumbtable_t *table = (dt_thumbtable_t *)user_data;
-  int delta;
 
+   // Handle continuous scrolling separately
+  if(table->mode == DT_THUMBTABLE_MODE_FILEMANAGER && e->direction == GDK_SCROLL_SMOOTH
+    && !((e->state & GDK_CONTROL_MASK) == GDK_CONTROL_MASK))
+  {
+    gdouble delta;
+    if (dt_gui_get_scroll_deltas(e, NULL, &delta))
+    {
+      // Scale delta to represent scroll size in pixels.
+      // On macOS reverses scaling in dt_gui_get_scroll_deltas
+      delta = delta * 100;
```

Comment:
And on other OSes what does this do?..

Reply:
I haven't tested other OSes. Scaling in _dt_gui_get_scroll_deltas()_ makes macOS work with other controls (not part of this PR). So I assumed that made all OSes work similarly. Definitely needs testing on other systems.

I think using scroll deltas directly is how scrolling is intended to work. That is why I added the comment.